core/state_update.go

```diff
@@ -35,6 +35,21 @@ func EmptyStateDiff() *StateDiff {
 	}
 }
 
+func (d *StateDiff) Length() uint64 {
+	var length uint64
+
+	for _, storageDiff := range d.StorageDiffs {
+		length += uint64(len(storageDiff))
+	}
+	length += uint64(len(d.Nonces))
+	length += uint64(len(d.DeployedContracts))
+	length += uint64(len(d.DeclaredV0Classes))
+	length += uint64(len(d.DeclaredV1Classes))
+	length += uint64(len(d.ReplacedClasses))
```

Comment:
maybe we should make `length` var as int and cast it into `uint64` at the bottom?
I don't think that in practice sum of all these fields will reach 2^32 (max value of int)

Reply:
Done